Makefile

```diff
@@ -61,45 +51,14 @@ all: $(TARGET)
 # FPGA
 # -----------------------------------------------------------------------------
 
-# FIRRTL compiler
-FIRRTL_VERSION="1.2.2"
-FIRRTL_JAR = $(FPGA_DIR)/firrtl.jar
-
-$(FIRRTL_JAR):
-	wget https://github.com/freechipsproject/firrtl/releases/download/v$(FIRRTL_VERSION)/firrtl.jar -O $(FIRRTL_JAR)
-
-# Alias for $(FIRRTL)
-firrtl_jar: $(FIRRTL_JAR)
-
-# Raw (hi-)firrtl generated by Chisel.
-FIRRTL_RAW = build/$(MODULE).hi.fir
-firrtl_raw: $(FIRRTL_RAW)
-
 # Raw Verilog generated from FIRRTL.
 VERILOG_RAW = build/$(MODULE).raw.v
 verilog_raw: $(VERILOG_RAW)
 
-$(FIRRTL_RAW): $(SRC_DIR)/$(MODULE)/*.scala $(FIRRTL_JAR) $(MILL_BIN)
-	$(MILL_BIN) flexpret.run "$(CORE_CONFIG)" --compiler high --target-dir "build/"
-	# high-firrtl is dumped into $(FIRRTL_RAW)
```

Comment:
Ditto this if possible as I use the FIRRTL for my own research more than Verilog.

Reply:
Same here.